yt_dlp/extractor/dropbox.py

```diff
@@ -12,7 +12,7 @@
 
 
 class DropboxIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?dropbox[.]com/sh?/(?P<id>[a-zA-Z0-9]{15})/.*'
+    _VALID_URL = r'https?://(?:www\.)?dropbox\.com/(?:scl/fi/|e/scl/fi/|s/|sh/)(?P<id>[^/]+)/.*'
```

Comment:
isn't `\w+` enough for the `id`?
```suggestion
    _VALID_URL = r'https?://(?:www\.)?dropbox\.com/(?:(?:e/)?scl/fi|sh?)/(?P<id>\w+)/.*'
```
or else this:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?dropbox\.com/(?:(?:e/)?scl/fi|sh?)/(?P<id>[^/#?]+)/.*'
```